content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,281 @@
+
+## Roles
+
+The following sections covers the roles that are commonly involved in the
+operations of the EGI Infrastructure. The correspondent
+[roles defined in Configuration Database (GOCDB)](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Roles)
+give specific rights in the Configuration Database itself and in other EGI
+services. Other terms and definitions can be found in the
+[EGI Glossary](https://go.egi.eu/glossary).
+
+### Resource Centre level
+
+#### Resource Centre Administrator
+
+An individual responsible for installing, operating, maintaining and supporting
+one or more Resources or IT Services in a Resource Centre. In the scope of
+Operations, Resource Centres (RC) administrators primarily receive and react on
+incidents at their Resource Centre and on service requests notified through
+tickets created on the [EGI Helpdesk service](../../../internal/helpdesk). They
+should respond to the tickets in a suitable time frame as defined in the
+[Resource Centre Operational Level Agreement (OLA)](https://documents.egi.eu/document/31)
+and be aware of the alarms at their site, eg. through the
+[Operations Dashboard](https://operations-portal.egi.eu). Sites must only
+operate supported middleware versions. This implies upgrading it regularly.
+Emergency releases are treated in a special way. Resource Centre Administrators
+must react to security issues that are at a global level, but affect their site.
+See
+[SEC03 EGI-CSIRT Critical Vulnerability Handling](https://wiki.egi.eu/wiki/SEC03_EGI-CSIRT_Critical_Vulnerability_Handling).
+
+#### Resource Centre Operations Manager
+
+An individual who leads the Resource Centre operations, who is the official
+technical contact person in the connected organisation, and who is locally
+supported by a team of Resource Centre Administrators. The Resource Centre
+Operations Manager is responsible for the site at the political and legal levels
+and for signing the
+[Operational Level Agreement](https://documents.egi.eu/public/ShowDocument?docid=31))
+between the Resource Centre and its hosting NGI. The Resource Centre Operations
+Manager is also responsible for enforcing the EGI policies and procedures by the
+Resource Centre and for assigning and approving the other site roles in the
+Configuration Database. Further, they should ensure that administrators are
+subscribed to the relevant mailing lists.
+
+#### Resource Centre Security Officer
+
+The person responsible for keeping the site compliant with the
+[EGI security policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies). They are
+the primary contact for the NGI Security officer and
+[EGI Computer Security Incident Response Team (CSIRT)](https://csirt.egi.eu).
+The Site Security Officer deals with security incidents and shall respond to
+enquiries in a timely fashion as defined in the collection of
+[security procedures and policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies).
+
+### Regional level
```

Comment:
@paolini78, @mviljoen-egi  maybe an introductory text on describing what means regional level would help.

Reply:
I have updated the text in the Roles section.